### PR TITLE
chore: Upgrade remaining bigtable samples - issue #1389

### PR DESCRIPTION
**Pull request by saranshdhingra**

### CODE REVIEW

**Comment by saranshdhingra** [comment]
The extra tag removed out of the 19 deleted region tags(bigtable_filters_print) isn't used in any page in the docs for any language.

File: bigtable/src/filter_composing_chain.php [hunk 1]
```diff
@@ -48,20 +48,17 @@ function filter_composing_chain(
...
-// Helper function for printing the filtered data
-function read_filter($table, $filter)
-{
+    
     $rows = $table->readRows([
         'filter' => $filter
     ]);
 
     foreach ($rows as $key => $row) {
+        // helper function defined in https://cloud.google.com/bigtable/docs/samples/bigtable-reads-print
```
Comment (bshaffer) [inline bigtable/src/filter_composing_chain.php:57]: Nit, but let's use a complete sentence here to be absolutely crystal clear:
```suggestion
        // The "print_row" helper function is defined in https://cloud.google.com/bigtable/docs/samples/bigtable-reads-print
```

File: bigtable/src/quickstart.php [hunk 2]
```diff
@@ -24,40 +22,39 @@
...
+// Read a row from my-table using a row key
+$row = $table->readRow($key);
 
-// The following 2 lines are only needed to run the samples
-require_once __DIR__ . '/../../testing/sample_helpers.php';
-\Google\Cloud\Samples\execute_sample(__FILE__, __NAMESPACE__, $argv);
+$columnFamilyId = 'cf1';
+$columnId = 'c1';
+// Get the Value from the Row, using the column_family_id and column_id
+
```
Comment (bshaffer) [inline bigtable/src/quickstart.php]: remove extra newline
```suggestion
```

File: bigtable/src/quickstart.php [hunk 3]
```diff
@@ -24,40 +22,39 @@
...
-
-    printf('Row key: %s' . PHP_EOL . 'Data: %s' . PHP_EOL, $key, $value);
-}
-// [END bigtable_quickstart]
+/** Uncomment and populate these variables in your code */
+// $projectId = 'The Google project ID';
+// $instanceId = 'The Bigtable instance ID';
+// $tableId = 'The Bigtable table ID';
+
+
```
Comment (bshaffer) [inline bigtable/src/quickstart.php]: Remove extra newline
```suggestion
```

**Review by bshaffer — COMMENTED** [review]
One suggestion (which applies to each sample), a few style nits, and this looks really good!

**Review by bshaffer — APPROVED** [review]

File: bigtable/src/filter_limit_cells_per_col.php [hunk 4]
```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Google\Cloud\Samples\Bigtable;
+
+/**
+ * Copyright 2019 Google LLC.
```
Comment (bshaffer) [inline bigtable/src/filter_limit_cells_per_col.php:6]: The copyright years need updated on all these files!

File: bigtable/src/filter_limit_cells_per_col.php [hunk 5]
```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Google\Cloud\Samples\Bigtable;
```
Comment (bshaffer) [inline bigtable/src/filter_limit_cells_per_col.php:3]: Namespace should go just above the region tags, to be consistent with our other samples

File: bigtable/src/filter_limit_col_family_regex.php [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/bigtable/README.md
+ */
+
+// [START bigtable_filters_limit_col_family_regex]
+use Google\Cloud\Bigtable\BigtableClient;
+use Google\Cloud\Bigtable\Filter;
+
+/**
+ * Create a limiting filter on a column family with a regex
```
Comment (bshaffer) [inline bigtable/src/filter_limit_col_family_regex.php:32]: nit: lets put a newline between the description and `@param` tags
```suggestion
 * Create a limiting filter on a column family with a regex
 *
```

**Review by bshaffer — CHANGES_REQUESTED** [review]
Just a few nits I missed

**Comment by saranshdhingra** [comment]
> Just a few nits I missed. Also, do we have a plan for the snippet bot errors?

@bshaffer 

Namespace position: Updated in all bigtable samples
Line b/w description and `@param`: Updated in all bigtable samples
Copyright year change: Changed only in the new files(read snippets, write snippets, filter files, new samples added) as I imagine it's only for new files.

The CL for the new region tag paths is ready and as soon as this PR is merged we can proceed with that like the read and write snippets splitting.

**Review by bshaffer — APPROVED** [review]

**Pull request merged**